django_projects/locallibrary/catalog/templates/base_generic.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  {% block title %}<title>Local Library</title>{% endblock %}
```

Comment:
tìm hiểu i18n trong django để apply vào các templates cho các chuỗi hiển thị lên UI của end user nhé

Reply:
{% load i18n %} is needed for internationalization. The purpose of internationalization is to allow a single application to read in multiple languages. In order to do this: you need a few hooks called translation strings. To give your template access to these tags, put {% load i18n %} toward the top of your template

Reply:
Em apply I18n vào luôn nha, setting dùng 2 languages: (en-us, vi)